routing/turns_sound_settings.cpp

```diff
@@ -73,7 +72,23 @@ double Settings::ConvertUnitsToMeters(double distanceInUnits) const
       return MeasurementUtils::FeetToMeters(distanceInUnits);
   }
 
-  // m_lengthUnits is equal to LengthUnits::Undefined or to unknown value.
+  ASSERT(false, ());
+  return 0.;
+}
+
+double Settings::ConvertMetersToUnits(double distanceInMeters) const
+{
+  switch (m_lengthUnits)
+  {
+    case LengthUnits::Undefined:
```

Comment:
> Is it hard to refactor here?

I guess it'll be a PR with one commit of about 100 lines(addition/deletion). 

> Is it the only place where Undefined is used?

No. Of cause not. There are about 10 places where LengthUnits::Undefined is used. They shall be removed. 
Besides that there are some lines which shall be exchanged. Units::Metric -> LengthUnits::Meters and so on.

I guess we should do it but in a separate PR.